src/hotspot/share/memory/metaspace.hpp

```diff
@@ -113,8 +117,32 @@ class Metaspace : public AllStatic {
   static MetaWord* allocate(ClassLoaderData* loader_data, size_t word_size,
                             MetaspaceObj::Type type);
 
-  static bool contains(const void* ptr);
-  static bool contains_non_shared(const void* ptr);
+  // Returns true if the pointer points into class space, non-class metaspace, or the
+  // Klass portion of the CDS archive.
```

Comment:
This was the tiny comment nit.  This doesn't return the "Klass portion of the" CDS archive so can you remove those words?  Then it's good to go and I'll merge with it.  Thanks!

Reply:
Oh, I see.  How about "metadata portion" then?

Reply:
> Oh, I see. How about "metadata portion" then?

Okay, will do.